test/functional/ui/wildmode_spec.lua

```diff
@@ -30,3 +31,34 @@ describe("'wildmode'", function()
     end)
   end)
 end)
+
+describe('command line completion', function()
+  local screen
+
+  before_each(function()
+    clear()
+    screen = Screen.new(40, 5)
+    screen:attach()
+    screen:set_default_attr_ignore({{bold=true, foreground=Screen.colors.Blue}})
+  end)
+
+  after_each(function()
+    os.remove('Xtest-functional-viml-compl-dir')
```

Comment:
This works since there's nothing in the directory, but it reads oddly to me (partly due to unfamiliarity with Lua functions) to use `os.remove` here instead of the `helpers.rmdir` function.